src/types.ts

```diff
@@ -453,6 +453,7 @@ export type TigrisSchema<T> = {
 		type: TigrisDataTypes | TigrisSchema<unknown>;
 		primary_key?: TigrisPrimaryKey;
 		items?: TigrisArrayItem;
+		key?: TigrisPartitionKey;
```

Comment:
IMO: `key` is too generic. can we rename it to `partition_key`?

Reply:
@himank suggested `key` for the schema definition. I'm just matching it here.

Reply:
It is called `key` by other streaming solutions as well. Keeping it `key` should be okay.